app/views/shared/_flashes.html.erb

```diff
@@ -1,7 +1,9 @@
 <% flash.to_hash.slice(*ApplicationController::FLASH_KEYS).each do |type, message| %>
   <% if message.present? %>
-    <div class='alert <%= "alert-#{type}" %>' role='alert'>
-      <%= safe_join([message.html_safe]) %>
-    </div>
+    <%= render 'shared/alert', {
+      type: type,
+      message: safe_join([message.html_safe]),
```

Comment:
the safe_join on a manually HTML-safe'd string is confusing to me....? Do we store `<p>` tag or something inside our messages?

Reply:
> the safe_join on a manually HTML-safe'd string is confusing to me....? Do we store `<p>` tag or something inside our messages?

It was something I opted to keep unchanged from the previous (current) implementation:

https://github.com/18F/identity-idp/blob/538cf31d37a741b0c63126b5ec30cdf104682b46/app/views/shared/_flashes.html.erb#L4

But yeah, it does seem that the `safe_join` would be redundant now that you mention it 🤔

Reply:
> the safe_join on a manually HTML-safe'd string is confusing to me....? Do we store `<p>` tag or something inside our messages?

Removed redundant `safe_join` in e495e09de.